wallet-web/common/helpers.ts

```diff
@@ -110,15 +110,13 @@ export const validateIdentityKey = (key: string): boolean => {
   }
 }
 
-export const validateRawPort = (rawPort: string): boolean => {
-  // first of all it must be an integer
-  const port = parseInt(rawPort)
-  if (port == null) {
-    return false
-  }
-  // and it must be a non-zero 16 bit unsigned integer
-  return port >= 1 && port <= 65535
-}
+export const validateRawPort = (rawPort: number): boolean =>
+  typeof rawPort === 'number' &&
```

Comment:
the `typeof rawPort === 'number'` check is redundant, we know for sure it's a number since typescript wouldn't allow anything else here